examples/energymap/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+
+	gridintensity "github.com/thegreenwebfoundation/grid-intensity-go"
+	"github.com/thegreenwebfoundation/grid-intensity-go/energymap"
+)
+
+func main() {
+	token := os.Getenv("ENERGY_MAP_API_TOKEN")
```

Comment:
How can you get an API token for the Energy Map API?

Reply:
answer on main thread